it_zwierzak/adoption_announcement/views.py

```diff
@@ -1,22 +1,19 @@
 from django.shortcuts import render
 from django.views.generic import FormView
 from .forms import AdoptionForm
-from django.contrib.auth.decorators import login_required
 from django.contrib.auth.mixins import LoginRequiredMixin
+from .models import Announcement
 
 
-
-# zaadoptowane
 def adopted(request):
     return render(request, 'adoption_announcement/adopted_animals.html')
 
 
-# przeniesienie do strony z kafelkami adopcji
 def adoptions(request):
-    return render(request, 'adoption_announcement/animals_to_adoption.html')
+    announcement_data = Announcement.objects.all()
```

Comment:
skoro to widok, gdzie zwracamy listę `Annoucements` to widok powinien się nazywać `annoucements`, a nie adoptions, trochę mylące